packages/desktop/src/events/download/download.event.ts

```diff
@@ -1,13 +1,9 @@
 import { ipcMain } from 'electron'
```

Comment:
This is invalid, you cannot import electron directly, you need to use "window.require" because electron package is not on node_modules, it is injected by electron on the browser

Reply:
Please, double check the path to this file, this import from the *electron* module is being executed in electron's main process